docs/src/main/asciidoc/_elytron/Keycloak_SAML_Integration.adoc

```diff
@@ -0,0 +1,191 @@
+====
+Unlike the WildFly feature pack, this Keycloak feature pack is not part of a universe and so a fully
+qualified `group:artifact:version` reference to the feature pack is required.
+====
+
+From this point, applications can be configured as described in the
+https://www.keycloak.org/docs/latest/securing_apps/index.html#_jboss_adapter[Keycloak documentation].
+
+[[bootable-jar]]
+== Bootable JAR
+
+The next installation option is if you are creating a bootable JAR for your deployment and want to
+add the Keycloak SAML adapter to secure the deployment.  More details about bootable JAR support can
+be found in the <<Bootable_Guide.adoc#wildfly_bootable_JAR_development,Bootable JAR Guide>>.
+
+The following is an example plug-in configuration to create a bootable JAR for a web application
+using both the `web-server` and `keycloak-client-saml` layers:
+
+[source,xml,options="nowrap"]
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-jar-maven-plugin</artifactId>
+    <version>${version.wildfly.jar.maven.plugin}</version>
+    <configuration>
+        <feature-packs>
+            <feature-pack>
+                <location>wildfly@maven(org.jboss.universe:community-universe):current</location>
+            </feature-pack>
+            <feature-pack>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-adapter-galleon-pack</artifactId>
+                <version>999</version> <!-- TODO: Update version when we know what it is -->
+            </feature-pack>
+        </feature-packs>
+        <layers>
+            <layer>web-server</layer>
+            <layer>keycloak-client-saml</layer>
+        </layers>
+        <context-root>simple-webapp</context-root>
+        <cli-sessions>
+            <cli-session>
+                <script-files>
+                    <script>configure-saml.cli</script>
+                </script-files>
+            </cli-session>
+        </cli-sessions>
+    </configuration>
+    <executions>
+        <execution>
+            <goals>
+                <goal>package</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+----
+
+[NOTE]
+====
+As with using the CLI, the latest version of WildFly in the universe can be dynamically selected but the
+Keycloak feature pack requires the complete `groupId`, `artifactId`, and `version` to be specified.
+====
+
+The approaches to configure the web application are the same as described in the
+https://www.keycloak.org/docs/latest/securing_apps/index.html#_jboss_adapter[Keycloak documentation].
+In this example, a CLI script, `configure-saml.cli`, is executed to update the Keycloak SAML subsystem
+with relevant configuration.
+
+A sample script is shown below:
+
+[source]
+----
+/subsystem=keycloak-saml/secure-deployment=simple-webapp.war/:add
+
+/subsystem=keycloak-saml/secure-deployment=simple-webapp.war/SP="http://localhost:8090/simple-webapp"/:add(sslPolicy=EXTERNAL,logoutPage="logout"
```

Comment:
it looks like that the parenthesis opened with `:add` isn't closed: perhaps the statement is incomplete...
btw, I downloaded the adapter from https://www.keycloak.org/documentation but could not find such a script ...

Reply:
Thanks for catching that! I've added the closing parenthesis.

It's possible to download the subsystem commands from the Keycloak console as Krystof mentioned here:
https://github.com/wildfly/wildfly-proposals/pull/486#discussion_r1085144103